scripts/generate_export.py

```diff
@@ -9,7 +10,7 @@
 from solutions_toolkit.indico_wrapper import IndicoWrapper
 
 
-USAGE_STRING = "USAGE: python3 generate_export path/to/configuration_file"
+USAGE_STRING = "python3 generate_export path/to/configuration_file"
```

Comment:
Keep the USAGE bit, it's a clue to the user that it's an instruction. (Might be a good idea to use arg parse here, just a thought for refactoring).